_i18n/en.yml

```
@@ -763,3 +763,8 @@ blog:
 
 tags:
   notags: There are no posts for this tag.
+
+accessibility:
+  arrowup: "'Back to top' button"
```

A bit more natural when using a screenreader, and makes it more similar to what other sites do when describing links:

```suggestion
  arrowup: "Go back to the top of the page"
```